### PR TITLE
Feature/cu kqbfwv melee grunt nav mesh

### PR DESCRIPTION
**Pull request by isaacfdez**

### CODE REVIEW

**Comment by jcarlos0305** [comment]
Task linked: [CU-kqbfwv MeleeGrunt uses NavMesh](https://app.clickup.com/t/kqbfwv)

File: Penteract/Assets/Scripts/AIMovement.cpp [hunk 1]
```diff
@@ -22,6 +22,14 @@ GENERATE_BODY_IMPL(AIMovement);
 
 void AIMovement::Start() {
     player = GameplaySystems::GetGameObject(playerUID);
+    agent = GetOwner().GetComponent<ComponentAgent>();
+    if (agent) {
+        agent->SetMaxSpeed(maxSpeed);
+        agent->SetMaxAcceleration(9999);
+        agent->SetAgentObstacleAvoidance(true);
+        agent->RemoveAgentFromCrowd();
+        //agent->AddAgentToCrowd();
```
Comment (Jordi-Gil) [inline Penteract/Assets/Scripts/AIMovement.cpp:31]: Remove comment

File: Penteract/Assets/Scripts/AIMovement.cpp [hunk 2]
```diff
@@ -120,6 +130,8 @@ void AIMovement::OnAnimationFinished()
     else if (state == AIState::HURT && lifePoints <= 0) {
         animation->SendTrigger("HurtDeath");
         Debug::Log("Death");
+        //agent->SetMoveTarget(parentTransform->GetGlobalPosition());
```
Comment (Jordi-Gil) [inline Penteract/Assets/Scripts/AIMovement.cpp:133]: Remove comment

File: Penteract/Assets/Scripts/AIMovement.cpp [hunk 3]
```diff
@@ -44,6 +52,7 @@ void AIMovement::Update() {
             animation->SendTrigger("AttackHurt");
         }
         lifePoints -= damageRecieved;
+        //agent->SetMoveTarget(parentTransform->GetGlobalPosition());
```
Comment (Jordi-Gil) [inline Penteract/Assets/Scripts/AIMovement.cpp:55]: Remove comment

**Review by Jordi-Gil — APPROVED** [review]

**Review by Jordi-Gil — APPROVED** [review]

**Review by jcarlos0305 — APPROVED** [review]

**Review by Yonimevicio — APPROVED** [review]

**Pull request merged**